guidelines/sc/22/target-size-minimum.html

```diff
@@ -7,7 +7,7 @@ <h4>Target Size (Minimum)</h4>
     
     <p>The size of the <a>target</a> for <a>pointer inputs</a> is at least 24 by 24 <a>CSS pixels</a>, except where:</p>
     <ul>
-        <li><strong>Spacing:</strong> The <a>target offset</a> is at least 24 CSS pixels to every adjacent target;</li>
+        <li><strong>Spacing:</strong> The target is non-overlapping and has a <a>target offset</a> of at least 24 CSS pixels to every adjacent target;</li>
```

Comment:
I think that to avoid future "the target is overlapping the page background" comments we should specify what it is not overlapping. I think that we mean "The target does not overlap any other target and has a..."